### ISSUE TITLE
onnxruntime deploy onnx 

### ISSUE THREAD

**Issue by liufqing** [open-mmlab/mmdeploy#873]
[2022-08-07 21:17:44.507] [mmdeploy] [info] [model.cpp:95] Register 'DirectoryModel'
[2022-08-07 21:17:53.342] [mmdeploy] [error] [model.cpp:44] no ModelImpl can read model F:\DeepLearn\openmmlab\seg_test\panel_512x1024.onnx
[2022-08-07 21:17:53.342] [mmdeploy] [error] [model.cpp:15] load model failed. Its file path is 'F:\DeepLearn\openmmlab\seg_test\panel_512x1024.onnx'
[2022-08-07 21:17:53.355] [mmdeploy] [error] [model.cpp:20] failed to create model: not supported (2)
failed to create segmentor, code: 6

**Comment by RunningLeon** [comment]
@liufqing Hi, please follow the[ template ](https://github.com/open-mmlab/mmdeploy/blob/master/.github/ISSUE_TEMPLATE/bug-report.md)to offer us the full script and env info to reproduce.

**Comment by liufqing** [comment]
env:
# Name                    Version                   Build  Channel
addict                    2.4.0                    pypi_0    pypi
anyio                     3.6.1                    pypi_0    pypi
argon2-cffi               21.3.0                   pypi_0    pypi
argon2-cffi-bindings      21.2.0                   pypi_0    pypi
asynctest                 0.13.0                     py_0    http://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
atomicwrites              1.4.1                    pypi_0    pypi
attrs                     21.4.0             pyhd3eb1b0_0    http://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
babel                     2.10.3                   pypi_0    pypi
backcall                  0.2.0                    pypi_0    pypi
beautifulsoup4            4.11.1                   pypi_0    pypi
black                     22.6.0                   pypi_0    pypi
blas                      1.0                         mkl    http://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/free
bleach                    5.0.1                    pypi_0    pypi
brotlipy                  0.7.0           py37h2bbff1b_1003    http://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
ca-certificates           2022.07.19           haa95532_0    http://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
certifi                   2022.6.15        py37haa95532_0    http://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
cffi                      1.15.1                   pypi_0    pypi
charset-normalizer        2.0.4              pyhd3eb1b0_0    http://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
click                     8.1.3                    pypi_0    pypi
codecov                   2.1.11             pyhd3eb1b0_0    http://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
colorama                  0.4.5            py37haa95532_0    http://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
commonmark                0.9.1                    pypi_0    pypi
coverage                  6.3.2            py37h2bbff1b_0    http://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
cryptography              37.0.1           py37h21b164f_0    http://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
cudatoolkit               10.2.89              h74a9793_1    http://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
cudnn                     7.6.5                cuda10.2_0    http://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
cycler                    0.11.0                   pypi_0    pypi
debugpy                   1.6.2                    pypi_0    pypi
decorator                 5.1.1                    pypi_0    pypi
deprecation               2.1.0                    pypi_0    pypi
descartes                 1.1.0                    pypi_0    pypi
dill                      0.3.5.1                  pypi_0    pypi
entrypoints               0.4                      pypi_0    pypi
fastjsonschema            2.16.1                   pypi_0    pypi
fire                      0.4.0                    pypi_0    pypi
flake8                    4.0.1                    pypi_0    pypi
fonttools                 4.33.3                   pypi_0    pypi
freetype                  2.10.4               hd328e21_0    http://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
h5py                      3.7.0                    pypi_0    pypi
idna                      3.3                pyhd3eb1b0_0    http://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
importlib-metadata        4.12.0                   pypi_0    pypi
importlib-resources       5.9.0                    pypi_0    pypi
importlib_metadata        4.11.3               hd3eb1b0_0    http://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
iniconfig                 1.1.1              pyhd3eb1b0_0    http://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
intel-openmp              2021.4.0          haa95532_3556    http://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
ipykernel                 6.15.1                   pypi_0    pypi
ipython                   7.34.0                   pypi_0    pypi
ipython-genutils          0.2.0                    pypi_0    pypi
ipywidgets                7.7.1                    pypi_0    pypi
isort                     5.9.3              pyhd3eb1b0_0    http://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
jedi                      0.18.1                   pypi_0    pypi
jinja2                    3.1.2                    pypi_0    pypi
joblib                    1.1.0                    pypi_0    pypi
jpeg                      8d                            0    http://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/free
json5                     0.9.8                    pypi_0    pypi
jsonschema                4.7.2                    pypi_0    pypi
jupyter                   1.0.0                    pypi_0    pypi
jupyter-client            7.3.4                    pypi_0    pypi
jupyter-console           6.4.4                    pypi_0    pypi
jupyter-core              4.11.1                   pypi_0    pypi
jupyter-packaging         0.12.2                   pypi_0    pypi
jupyter-server            1.18.1                   pypi_0    pypi
jupyterlab                3.4.4                    pypi_0    pypi
jupyterlab-pygments       0.2.2                    pypi_0    pypi
jupyterlab-server         2.15.0                   pypi_0    pypi
jupyterlab-widgets        1.1.1                    pypi_0    pypi
kiwisolver                1.4.3                    pypi_0    pypi
libpng                    1.6.37               h2a8f88b_0    http://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
libtiff                   4.2.0                he0120a3_1    http://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
libuv                     1.40.0               he774522_0    http://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
libwebp                   1.2.2                h2bbff1b_0    http://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
llvmlite                  0.36.0                   pypi_0    pypi
lyft-dataset-sdk          0.0.8                    pypi_0    pypi
lz4-c                     1.9.3                h2bbff1b_1    http://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
markdown                  3.3.4                    pypi_0    pypi
markupsafe                2.1.1                    pypi_0    pypi
matplotlib                3.5.2                    pypi_0    pypi
matplotlib-inline         0.1.3                    pypi_0    pypi
mccabe                    0.6.1                    pypi_0    pypi
mistune                   0.8.4                    pypi_0    pypi
mkl                       2021.4.0           haa95532_640    http://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
mkl-service               2.4.0            py37h2bbff1b_0    http://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
mkl_fft                   1.3.1            py37h277e83a_0    http://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
mkl_random                1.2.2            py37hf11a4ad_0    http://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
mmcls                     0.23.1                   pypi_0    pypi
mmcv-full                 1.5.3                    pypi_0    pypi
mmdeploy                  0.6.0                     dev_0    <develop>
mmdet                     2.25.0                   pypi_0    pypi
mmdet3d                   1.0.0rc3                  dev_0    <develop>
mmsegmentation            0.25.0                    dev_0    <develop>
model-index               0.1.11                   pypi_0    pypi
multiprocess              0.70.13                  pypi_0    pypi
mypy-extensions           0.4.3                    pypi_0    pypi
nbclassic                 0.4.3                    pypi_0    pypi
nbclient                  0.6.6                    pypi_0    pypi
nbconvert                 6.5.0                    pypi_0    pypi
nbformat                  5.4.0                    pypi_0    pypi
nest-asyncio              1.5.5                    pypi_0    pypi
networkx                  2.2                      pypi_0    pypi
notebook                  6.4.12                   pypi_0    pypi
notebook-shim             0.1.0                    pypi_0    pypi
numba                     0.53.0                   pypi_0    pypi
numpy                     1.21.6                   pypi_0    pypi
numpy-base                1.21.5           py37hca35cd5_3    http://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
nuscenes-devkit           1.1.9                    pypi_0    pypi
onnx                      1.12.0                   pypi_0    pypi
onnx-simplifier           0.4.4                    pypi_0    pypi
onnxruntime               1.8.1                    pypi_0    pypi
open3d                    0.15.1                   pypi_0    pypi
opencv-python             4.6.0.66                 pypi_0    pypi
openmim                   0.2.0                    pypi_0    pypi
openssl                   1.1.1q               h2bbff1b_0    http://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
ordered-set               4.1.0                    pypi_0    pypi
packaging                 21.3               pyhd3eb1b0_0    http://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
pandas                    1.3.5                    pypi_0    pypi
pandocfilters             1.5.0                    pypi_0    pypi
parso                     0.8.3                    pypi_0    pypi
pathspec                  0.9.0                    pypi_0    pypi
pickleshare               0.7.5                    pypi_0    pypi
pillow                    9.0.1            py37hdc2b20a_0    http://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
pip                       21.2.4           py37haa95532_0    http://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
platformdirs              2.5.2                    pypi_0    pypi
plotly                    5.9.0                    pypi_0    pypi
pluggy                    1.0.0            py37haa95532_1    http://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
plyfile                   0.7.4                    pypi_0    pypi
prettytable               3.3.0                    pypi_0    pypi
prometheus-client         0.14.1                   pypi_0    pypi
prompt-toolkit            3.0.30                   pypi_0    pypi
protobuf                  3.20.1                   pypi_0    pypi
psutil                    5.9.1                    pypi_0    pypi
py                        1.11.0             pyhd3eb1b0_0    http://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
pycocotools               2.0.4                    pypi_0    pypi
pycodestyle               2.8.0                    pypi_0    pypi
pycparser                 2.21               pyhd3eb1b0_0    http://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
pyflakes                  2.4.0                    pypi_0    pypi
pygments                  2.12.0                   pypi_0    pypi
pyopenssl                 22.0.0             pyhd3eb1b0_0    http://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
pyparsing                 3.0.9                    pypi_0    pypi
pyquaternion              0.9.9                    pypi_0    pypi
pyrsistent                0.18.1                   pypi_0    pypi
pysocks                   1.7.1                    py37_1    http://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
pytest                    7.1.2                    pypi_0    pypi
pytest-cov                3.0.0              pyhd3eb1b0_0    http://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
pytest-runner             5.3.1              pyhd3eb1b0_0    http://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
python                    3.7.13               h6244533_0    http://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
python-dateutil           2.8.2                    pypi_0    pypi
pytorch                   1.10.1          py3.7_cuda10.2_cudnn7_0    pytorch
pytorch-mutex             1.0                        cuda    pytorch
pywin32                   304                      pypi_0    pypi
pywinpty                  2.0.6                    pypi_0    pypi
pyyaml                    6.0                      pypi_0    pypi
pyzmq                     23.2.0                   pypi_0    pypi
qtconsole                 5.3.1                    pypi_0    pypi
qtpy                      2.1.0                    pypi_0    pypi
regex                     2022.6.2                 pypi_0    pypi
requests                  2.28.1           py37haa95532_0    http://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
rich                      12.5.1                   pypi_0    pypi
scikit-learn              1.0.2                    pypi_0    pypi
send2trash                1.8.0                    pypi_0    pypi
setuptools                61.2.0           py37haa95532_0    http://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
shapely                   1.8.2                    pypi_0    pypi
six                       1.16.0             pyhd3eb1b0_1    http://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
sniffio                   1.2.0                    pypi_0    pypi
soupsieve                 2.3.2.post1              pypi_0    pypi
sqlite                    3.38.5               h2bbff1b_0    http://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
tabulate                  0.8.10                   pypi_0    pypi
tenacity                  8.0.1                    pypi_0    pypi
tensorboard-plugin-wit    1.8.1                    pypi_0    pypi
termcolor                 1.1.0                    pypi_0    pypi
terminado                 0.15.0                   pypi_0    pypi
terminaltables            3.1.10                   pypi_0    pypi
threadpoolctl             3.1.0                    pypi_0    pypi
tinycss2                  1.1.1                    pypi_0    pypi
tk                        8.6.12               h2bbff1b_0    http://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
toml                      0.10.2             pyhd3eb1b0_0    http://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
tomli                     2.0.1            py37haa95532_0    http://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
tomlkit                   0.11.1                   pypi_0    pypi
torchaudio                0.10.1               py37_cu102    pytorch
torchvision               0.11.2               py37_cu102    pytorch
tornado                   6.2                      pypi_0    pypi
tqdm                      4.64.0                   pypi_0    pypi
traitlets                 5.3.0                    pypi_0    pypi
trimesh                   2.35.39                  pypi_0    pypi
typed-ast                 1.5.4                    pypi_0    pypi
typing_extensions         4.1.1              pyh06a4308_0    http://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
urllib3                   1.26.9           py37haa95532_0    http://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
vc                        14.2                 h21ff451_1    http://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
vs2015_runtime            14.27.29016          h5e58377_2    http://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
wcwidth                   0.2.5                    pypi_0    pypi
webencodings              0.5.1                    pypi_0    pypi
websocket-client          1.3.3                    pypi_0    pypi
wget                      3.2                      pypi_0    pypi
wheel                     0.37.1             pyhd3eb1b0_0    http://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
widgetsnbextension        3.6.1                    pypi_0    pypi
win_inet_pton             1.1.0            py37haa95532_0    http://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
wincertstore              0.2              py37haa95532_2    http://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
xz                        5.2.5                h8cc25b3_1    http://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
yapf                      0.32.0                   pypi_0    pypi
zipp                      3.8.0            py37haa95532_0    http://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
zlib                      1.2.12               h8cc25b3_2    http://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
zstd                      1.5.2                h19a0ad4_0    http://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main

I train a model using a netural network from stdc and then convert to a model in onnx format using tools.Calling under python can be used normally,but using onnxruntime to call an error.

**Comment by RunningLeon** [comment]
@liufqing pls. post full script of `image_segmentation`. here. BTW, only onnxruntime of cpu is supported in mmdeploy-sdk. Pls. make sure you are passing cpu to `device` argument.
